### ISSUE TITLE
Adding more training data causes LAS/UAS/Tag accuracy to suddenly nosedive

### ISSUE THREAD

**Issue by GruffPrys** [explosion/spaCy#1897]
I'm converting generated .conllu files to spaCy's .json format and using those to train a model from scratch for a new language (W elsh) using https://github.com/explosion/spaCy/blob/master/spacy/cli/train.py .

In spaCy 2.0.5, if the .conllu file has under ~1100 sentences, the training works relatively w ell considering the lack of data:

```
"name":"1050_words",
  "accuracy":{
    "ents_f":0.0,
    "uas":95.3439153439,
    "las":95.3439153439,
    "token_acc":100.0,
    "ents_r":0.0,
    "tags_acc":65.7142857143,
    "ents_p":0.0
  }
```  
Here, the accuracy was still increasing after 49 iterations.

However, adding a few more sentences to the training data causes the training to flatline on the first iteration at around 5% for tags_acc and 25% for uas/las, without any improvement whatsoever on further epochs.

```  
"name": "1100_words",
"accuracy": {
        "ents_f": 0.0,
        "uas": 25.0292397661,
        "ents_p": 0.0,
        "las": 25.0292397661,
        "tags_acc": 5.2436647173,
        "token_acc": 100.0,
        "ents_r": 0.0
    },
``` 

I was experiencing the same problem with spaCy 1.8.2, but the treshold of failure was around 12,000 sentences (10x higher), so I was able to produce more accurate models with the earlier version.

Is there something obvious that I'm doing wrong? Should I be changing any specific hyperparamaters to overcome this? After reading the docs I've tried increasing the dropout and batch size but unfortunately the problem remains.
 
If it's relevant, I'm generating the .conllu files from sentence templates into which I'm placing random instances of the appropriate noun, adjective etc. There are between 4 and 6 words in each sentence, and the appropriate id, head, and morphological features are calculated and applied by my scripts (my aim here is to investigate whether spaCy's parser alone can deal with [Initial Consonant Mutation](https://en.wikipedia.org/wiki/Consonant_mutation#Celtic_languages) or whether an alternative approach will be required). I currently have 6 different sentence templates, all of which are reasonably similar in terms of syntax. All the sentences are shuffled prior to training. 
 
Any suggestions gratefully received!

* Operating System:  Ubuntu 16.04
* Python Version Used:  3.5
* spaCy Version Used: 1.8.2 and 2.0.5
* Environment Information:

**Comment by honnibal** [comment]
I think this is almost certainly an artifact of your experiment. These sorts of things can easily happen when you're working from generated text.

**Comment by GruffPrys** [comment]
Thanks, I'd assumed as much and have been annotating more realistic texts in the meantime in the hope of combining both approaches.

The models did work surprisingly well (even with so few training sentences) but they may never be as accurate as hand-crafted rules in this particular use case. We'll see.
